tests/utils.py

```diff
@@ -58,6 +60,10 @@ def create_send_est():
     payload = TaxDeclarationPayload(est_data=create_form_data(), meta_data=create_meta_data())
     return TaxDeclarationDto(payload=payload, clientIdentifier="steuerlotse")
 
+def create_send_grundsteuer():
+    return GrundsteuerDto(payload=SampleGrundsteuerData().parse(), clientIdentifier="steuerlotse")
```

Comment:
Although it is only for testing, the client identifier should not be "steuerlotse" right? But "grundsteuer"?